### PR TITLE
[Spanish] localize 'Debugging

### PR DESCRIPTION
**Pull request by vasucp1207**
Signed-off-by: Vasu Singh <vasucp1207@gmail.com>

### Describe your changes
[Spanish] localize 'Debugging

### Checklist before opening this PR (put `x` in the checkboxes)
- [X] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [X] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by raelga — APPROVED** [review]

File: content/es/debugging.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+título: Depuración
+Estado: COMPLETADO
+categoría: concepto
+etiquetas: ["aplicación", "", ""]
```
Comment (electrocucaracha) [inline content/es/debugging.md:5]:
```suggestion
title: Depuración
status: Completed
category: concepto
tags: ["application", "", ""]
```

File: content/es/debugging.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+---
+
+## Lo que es
```
Comment (electrocucaracha) [inline content/es/debugging.md:8]:
```suggestion
## ¿Qué es?
```

File: content/es/debugging.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+La depuración es el proceso o actividad de encontrar y resolver errores (bugs) de programas, software o sistemas informáticos para obtener el resultado deseado.
+Un error es un defecto o un problema que conduce a resultados incorrectos o inesperados.
+
+## Problema que aborda
+
+El desarrollo de software es una actividad compleja que hace que sea casi imposible escribir código sin introducir errores.
+Esos errores conducen a un código que probablemente no funcionará como se desea (comportamiento indefinido) cuando se ejecute.
+Dependiendo de cuán crítica sea una aplicación, los errores pueden tener un impacto negativo significativo, financieramente o incluso en vidas humanas.
```
Comment (electrocucaracha) [inline content/es/debugging.md:17]:
```suggestion
Dependiendo de que tan crítica sea una aplicación, los bugs pueden tener un impacto negativo significativo; financiero o incluso en vidas humanas.
```

File: content/es/debugging.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+Por lo general, el código de la aplicación tiene que pasar por diferentes etapas o entornos en los que se prueba.
+Cuanto más crítica es una aplicación, más precisa debe ser la prueba.
+
+## Cómo ayuda
```
Comment (electrocucaracha) [inline content/es/debugging.md:21]:
```suggestion
## ¿Cómo ayuda?
```

File: content/es/debugging.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+Cuando aparecen errores, los ingenieros tienen que depurar (p. ej., encontrar y reparar) la aplicación para disminuir el comportamiento no deseado de los sistemas de producción.
+La depuración no es una tarea fácil, ya que los ingenieros deben rastrear el origen del comportamiento no deseado.
+Requiere conocimiento sobre el código en sí y el contexto de ejecución en tiempo de ejecución.
+Aquí es donde las diferentes técnicas y herramientas de depuración resultan útiles.
+El análisis de registros, seguimientos y métricas, por ejemplo, se utiliza para la depuración directamente en producción.
```
Comment (electrocucaracha) [inline content/es/debugging.md:27]:
```suggestion
El análisis de registros, trazas y métricas, son un ejemplo de lo que se utiliza para la depuración directamente en producción.
```

File: content/es/debugging.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
+---
+título: Depuración
+Estado: COMPLETADO
+categoría: concepto
+etiquetas: ["aplicación", "", ""]
+---
+
+## Lo que es
+
+La depuración es el proceso o actividad de encontrar y resolver errores (bugs) de programas, software o sistemas informáticos para obtener el resultado deseado.
```
Comment (electrocucaracha) [inline content/es/debugging.md:10]:
```suggestion
La depuración es el proceso o actividad de encontrar y resolver bugs (o errores) de programas, software o sistemas informáticos para obtener el resultado deseado.
```

File: content/es/debugging.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+Un error es un defecto o un problema que conduce a resultados incorrectos o inesperados.
```
Comment (electrocucaracha) [inline content/es/debugging.md:11]:
```suggestion
Un bug es un defecto o un problema que conduce a resultados incorrectos o inesperados.
```

File: content/es/debugging.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+
+## Problema que aborda
+
+El desarrollo de software es una actividad compleja que hace que sea casi imposible escribir código sin introducir errores.
```
Comment (electrocucaracha) [inline content/es/debugging.md:15]:
```suggestion
El desarrollo de software es una actividad compleja que hace que sea casi imposible escribir código sin introducir bugs.
```

File: content/es/debugging.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+Esos errores conducen a un código que probablemente no funcionará como se desea (comportamiento indefinido) cuando se ejecute.
```
Comment (electrocucaracha) [inline content/es/debugging.md:16]:
```suggestion
Esos bugs conducen a un código que probablemente no funcionará como se desea (un comportamiento no definido) cuando se ejecute.
```

File: content/es/debugging.md [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+Dependiendo de cuán crítica sea una aplicación, los errores pueden tener un impacto negativo significativo, financieramente o incluso en vidas humanas.
+Por lo general, el código de la aplicación tiene que pasar por diferentes etapas o entornos en los que se prueba.
```
Comment (electrocucaracha) [inline content/es/debugging.md:18]:
```suggestion
Por lo general, el código de la aplicación tiene que pasar por diferentes etapas o ambientes en los que se prueba.
```

File: content/es/debugging.md [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+Cuanto más crítica es una aplicación, más precisa debe ser la prueba.
+
+## Cómo ayuda
+
+Cuando aparecen errores, los ingenieros tienen que depurar (p. ej., encontrar y reparar) la aplicación para disminuir el comportamiento no deseado de los sistemas de producción.
```
Comment (electrocucaracha) [inline content/es/debugging.md:23]:
```suggestion
Cuando aparecen bugs, los ingenieros tienen que depurar (p. ej., encontrar y reparar) la aplicación para disminuir el comportamiento no deseado de los sistemas de producción.
```

File: content/es/debugging.md [hunk 12]
```diff
@@ -0,0 +1,29 @@
...
+La depuración no es una tarea fácil, ya que los ingenieros deben rastrear el origen del comportamiento no deseado.
+Requiere conocimiento sobre el código en sí y el contexto de ejecución en tiempo de ejecución.
+Aquí es donde las diferentes técnicas y herramientas de depuración resultan útiles.
+El análisis de registros, seguimientos y métricas, por ejemplo, se utiliza para la depuración directamente en producción.
+Los desarrolladores pueden usar la depuración interactiva para recorrer el código en tiempo de ejecución mientras analizan el contexto de ejecución relacionado.
+Una vez que han identificado la fuente de la falla, corrigen el código y crean una corrección de errores o un parche.
```
Comment (electrocucaracha) [inline content/es/debugging.md:29]:
```suggestion
Una vez que han identificado la fuente de la falla, corrigen el código y crean un bug fix o parche.
```

**Review by electrocucaracha — CHANGES_REQUESTED** [review]

File: content/es/debugging.md [hunk 13]
```diff
@@ -0,0 +1,29 @@
...
+
+La depuración es el proceso o actividad de encontrar y resolver errores (bugs) de programas, software o sistemas informáticos para obtener el resultado deseado.
+Un error es un defecto o un problema que conduce a resultados incorrectos o inesperados.
+
+## Problema que aborda
+
+El desarrollo de software es una actividad compleja que hace que sea casi imposible escribir código sin introducir bugs.
+Esos bugs conducen a un código que probablemente no funcionará como se desea (un comportamiento no definido) cuando se ejecute.
+Dependiendo de que tan crítica sea una aplicación, los bugs pueden tener un impacto negativo significativo; financiero o incluso en vidas humanas.
+Por lo general, el código de la aplicación tiene que pasar por diferentes etapas o ambientes en los que se prueba.
```
Comment (92nqb) [inline content/es/debugging.md:18]:
```suggestion
Por lo general, el código de una aplicación tiene que pasar por diferentes etapas o entornos en los que se prueba.
```

File: content/es/debugging.md [hunk 14]
```diff
@@ -0,0 +1,29 @@
...
+
+## ¿Cómo ayuda?
+
+Cuando aparecen bugs, los ingenieros tienen que depurar (p. ej., encontrar y reparar) la aplicación para disminuir el comportamiento no deseado de los sistemas de producción.
+La depuración no es una tarea fácil, ya que los ingenieros deben rastrear el origen del comportamiento no deseado.
+Requiere conocimiento sobre el código en sí y el contexto de ejecución en tiempo de ejecución.
+Aquí es donde las diferentes técnicas y herramientas de depuración resultan útiles.
+El análisis de registros, trazas y métricas, son un ejemplo de lo que se utiliza para la depuración directamente en producción.
+Los desarrolladores pueden usar la depuración interactiva para recorrer el código en tiempo de ejecución mientras analizan el contexto de ejecución relacionado.
+Una vez que han identificado la fuente de la falla, corrigen el código y crean un bug fix o parche.
```
Comment (92nqb) [inline content/es/debugging.md:29]:
```suggestion
Una vez que han identificado el origen del error, corrigen el código y crean un bug fix o parche.
```

**Review by 92nqb — CHANGES_REQUESTED** [review]
@vasucp1207 Thanks for contributing, I've added few comments

**Review by electrocucaracha — APPROVED** [review]

**Review by 92nqb — APPROVED** [review]
lgtm

**Pull request merged**